src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -218,6 +222,19 @@ public void run() {
         .build();
   }
 
+  private OutputView.CommentView commentView(List<String> configs, MethodContext context) {
+    String comment = configs.get(0);
+    Object[] args =
+        configs
+            .subList(1, configs.size())
+            .stream()
+            .map(c -> context.getNamer().localVarName(Name.anyLower(c)))
+            .toArray(Object[]::new);
+    return OutputView.CommentView.newBuilder()
+        .lines(ImmutableList.<String>of(String.format(comment, args)))
```

Comment:
This only generates one line in `OutputView.CommentView`, right?  How do we deal with user-specified `\n`s in the comment? Seems to me we would want to split on those so we have separate lines in the comment.

Also, how do we deal with extra-long lines specified by the user? Do we rely on a post-formatter to adjust them?